purchase_order_line_stock_available/views/purchase_view.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="purchase_order_form_view_stock" model="ir.ui.view">
+        <field name="name">purchase.order.form.view.stock</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='order_line']/tree//field[@name='qty_invoiced']"
+                position="after"
+            >
+                <field name="virtual_available" optional="show" />
```

Comment:
To avoid a very large column in the view:

```suggestion
                <field name="virtual_available" optional="show" string="Forecasted" />
```